README.md

```diff
@@ -188,6 +184,21 @@ It is hard knowing what is going on inside of some of these ADTs or your wonderf
 #### `tryCatch : (a -> b) -> a -> Either e b`
 Typical try-catch blocks are very imperative in their usage. This `tryCatch` function provides a means of capturing that imperative nature in a simple declarative style. Pass it a function that could fail and it will return you another function wrapping the first function. When called the new function will either return the result in an `Either.Right` if everything was good, or an error wrapped in an `Either.Left` if it fails.
 
+### Logic Functions
+The functions in this section are used to represent simple to complex logical branching in a declarative manner. All of these function will take predicate functions or `Pred`s and either return a resulting `Boolean` or run some function based on the result of the predicates. All of these functions work with functions/`Pred`s instead of values to allow for building out complex, "lazy" evaluated and allow easy composition.
```

Comment:
Well they work with both predicate functions, functions in the form of `(a -> Boolean)` and also with the ADT `Pred`. Those are two distinct things.

So I like that rewording for that first sentence. Will update.
So are you saying that the typical developer would not know what a predicate is?